cpu/arm7_common/periph/pm.c

```diff
@@ -8,6 +8,7 @@
 
 /**
  * @ingroup     cpu_arm7_common
+ * @ingroup     drivers_periph_pm
```

Comment:
On the other, as you already said, I don't know how much it matters for C files. But then again, why not have proper grouping? In case of cpu specific periph drivers it makes sense to have both groups in place?